doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -17,7 +17,7 @@ command:
 include::../protos/vkGetInstanceProcAddr.txt[]
 
   * pname:instance is the instance that the function pointer will be
-    compatible with.
+    compatible with, or `VK_NULL_HANDLE` for commands not dependent on any instance.
```

Comment:
I done research on the `NULL` vs `VK_NULL_HANDLE`:

The generator script specifically chooses NULL for dispatchable handles:
https://github.com/KhronosGroup/Vulkan-Docs/blob/5de77cf663c84cc2edfa355c49319dca0a869eb2/src/spec/generator.py#L1465

So not sure which is correct anymore... I guess the original author wanted them to have pointer-like semantics.

Oh well, it should be consistent, so either that needs to return back to `NULL` **or** rest of the spec needs to conform to this change.